packages/rsocket-core/src/Codecs.ts

```diff
@@ -1138,3 +1138,22 @@ function readPayload(
     frame.data = buffer.slice(offset, buffer.length);
   }
 }
+
+// exported as class to facilitate testing
+export class Deserializer {
```

Comment:
Essentially a proxy to the functional interfaces here in `Codec`, but useful to facilitate testing.